src/DssSpell.sol

```diff
@@ -15,23 +15,32 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 pragma solidity 0.6.12;
-
-// Enable ABIEncoderV2 when onboarding collateral
+// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
 // pragma experimental ABIEncoderV2;
+
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dapp/DSTokenAbstract.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
+interface RwaUrnLike {
+    function lock(uint256) external;
+    function draw(uint256) external;
+}
+
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
 
+    // TODO: update with the content of the executive vote.
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0114d88f41cd3cf49ba818d2b3c4f159490a6944/governance/votes/Executive%20vote%20-%20July%2013%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0000000000000000000000000000000000000000/governance/votes/ExecutiveXXX.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-07-13 MakerDAO Executive Spell | Hash: 0xb339556c10ffe677e6eac39f49444dcc65e5b7f993b824752adf9cb27afcf663";
+        "2022-07-27 MakerDAO Executive Spell | Hash: 0x0000000000000000000000000000000000000000000000000000000000000000";
 
-    uint256 constant MILLION = 10 ** 6;
+    address constant RWA_TOKEN_FAB = 0x2B3a4c18705e99bC29b22222dA7E10b643658552;
```

Comment:
Where is this code located?

Reply:
It looks like RWA-008 token has been sent to 0x03f1A14A5b31e2f1751b6db368451dFCEA5A0439 and RWA-009 has been sent to the Pause Proxy. What's the expected procedure here?

Reply:
https://github.com/clio-finance/mip21-toolkit/blob/master/src/tokens/RwaTokenFactory.sol